data/remnant/remnant ships.txt

```diff
@@ -1035,6 +1035,59 @@ ship "Puffin"
 
 
 
+ship "Smew"
+	sprite "ship/smew"
+	thumbnail "thumbnail/smew"
+	attributes
+		category "Fighter"
+		licenses
+			Remnant
+		"cost" 1641000
+		"shields" 1200
+		"hull" 2800
+		"required crew" 1
+		"bunks" 9
+		"mass" 60
+		"drag" 1.75
+		"heat dissipation" .65
+		"fuel capacity" 100
+		"cargo space" 90
+		"outfit space" 92
+		"weapon capacity" 10
+		"engine capacity" 46
+		"ramscoop" 0.8
+		"shield generation" 0.48
+		"shield energy" 0.32
+		"hull repair rate" 0.9
+		"hull energy" 0.58
```

Comment:
Actually, this would be a situation where it'd make a lot of sense for the ship to have extremely *poor* regeneration, but have shield and/or hull protection. As a life boat, it wouldn't be optimized for sustained combat (which is what regen is for), but it *would* be highly reinforced to take heavy hits, such as from the ship it is launching from blowing up. Perhaps even just hull protection. (shield protection I'd recon is a much more advanced tech than hull protection).

Reply:
No protection stats.